frontend/packages/client/src/hooks/useCommunity.js

```diff
@@ -143,12 +147,12 @@ export default function useCommunity({
             discordUrl,
             logo: communityLogo?.fileUrl,
             bannerImgUrl: communityBanner?.fileUrl,
-            contractAddress: setDefaultValue(
+            contractAddr: setDefaultValue(
               contractAddr,
-              '0x0ae53cb6e3f42a79'
```

Comment:
I think this could be an issue this address was from emulator, now this change uses addresses depending on the environment set for the FLOW_ENV

Reply:
as discussed, if this is going to use any address we will have to come up with a different method for this.